packages/adp-tooling/src/types.ts

```diff
@@ -35,21 +35,41 @@ export interface AdpWriterConfig {
     target: AbapTarget;
     ui5?: {
         minVersion?: string;
+        ui5Version?: string;
+        ui5EndpointUrl?: string;
```

Comment:
Since these are under `ui5` maybe we can omit this prefix `ui5`? So it becomes this:
```suggestion
        version?: string;
        endpointUrl?: string;
```
Or just `url`?

Reply:
Done